env/.nginx.env.example

```diff
@@ -0,0 +1 @@
+NGINX_HOST=localhost
```

Comment:
Why do we need this, and how does it have to look like for staging / production? Our domain? Or still localhost?

Reply:
Currently this does not get changed. On staging and production this should be `usedgoodreads.com`, but that has other problems. See comment below